src/main/java/roomescape/domain/member/domain/Role.java

```diff
@@ -0,0 +1,11 @@
+package roomescape.domain.member.domain;
+
+public enum Role {
+
+    MEMBER, ADMIN,
+    ;
```

Comment:
사소한 부분이긴한데 이런것도 컨벤션입니다.
같이 작업할 때 이런 요소들이 conflict을 유발하기도 합니다.

```suggestion
    MEMBER, 
    ADMIN,
    ;
```